en/style/typography.md

```diff
@@ -1,20 +1,24 @@
+There are 13 text (character) styles in the **Indigo.Design System** that constitute its typography and let you set up your theme's typeface. Although we are using [Titillium Web](https://fonts.google.com/specimen/Titillium+Web) as a default, we really want to give every designer the flexibility to craft his applications with style. Indigo.Design follows the way typography is defined in the [Material Type System](https://material.io/design/typography/the-type-system.html#type-scale) and complies with the implementation of the [Ignite UI for Angular Typography](https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes/typography).
 
 <img class="responsive-img" src="../images/typography_default.png" srcset="../images/typography_default@2x.png 2x" />
+<div class="divider--half"></div>
+<div class="divider--half"></div>
 
-In Sketch, these can be found on a separate page named `🎨 Typography` and are available as `Text Styles` that can be applied to any text element. In Adobe XD they are placed on an Artboard with the same name in the left column of the library file and exist as `Character Styles` that are used across components and patterns.
+- In **Figma**, all text styles can be found on a separate page named `↳ Typography` which is part of the **Indigo.Design Sytem** `🧱 Foundation`. The text styles are available as `Text Styles` that can be applied to any text element.
+- In **Sketch**, these can be found on a separate page named `🎨 Typography` and are available as `Text Styles` that can be applied to any text element.
+- In **Adobe XD** they are placed on an Artboard with the same name in the left column of the library file and exist as `Character Styles` that are used across components and patterns.
 
 ## Typography Styles
 
-To use the typography system in your project, all you need to do is apply one of the text (character) styles defined in the library to a native text element. Sketch offers a vast selection of type presets with different sizes, colors, and alignments, while Adobe XD defines only the size and lets you adjust the alignment and apply a color style.
+To use the typography system in your project, all you need to do is apply one of the text (character) styles defined in the library to a native text element. Sketch offers a vast selection of type presets with different sizes, colors, and alignments. Figma defines the Size, Weight, Line height, and Ratio (Letter spacing) and lets you adjust the alignment and apply a color style, while in Adobe XD, the text styles come only with predefined size and let you adjust the alignment and apply a color style.
 
 - 6 Headlines from `H1` to `H6` for titles.
 - 2 `Subtitle` sizes.
```

Comment:
I believe we have more styles - the hyperlink styles + detail 1, detail 2

Reply:
As far as I see, they are in the internal fonts folder in Sketch, so my assumption was that we don't need to list them here, but if I am wrong, I will edit them. Also, if the mentioned styles are part of the official typography, I believe that we need to add them to the Typography page as well.